StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Vertical Main Screen/RowButton.swift

```diff
@@ -473,7 +473,12 @@ extension RowButton {
         let imageView = UIImageView(image: paymentMethod.makeSavedPaymentMethodRowImage())
         imageView.contentMode = .scaleAspectFit
         let button = RowButton.create(appearance: appearance, type: .saved(paymentMethod: paymentMethod), imageView: imageView, text: paymentMethod.paymentSheetLabel, subtext: subtext, badgeText: badgeText, accessoryView: accessoryView, isEmbedded: isEmbedded, didTap: didTap)
-        button.accessibilityHelperView.accessibilityLabel = paymentMethod.paymentSheetAccessibilityLabel
+        button.accessibilityHelperView.accessibilityLabel = {
+            if let badgeText {
+                return (paymentMethod.paymentSheetAccessibilityLabel ?? "") + ", \(badgeText)"
+            }
+            return paymentMethod.paymentSheetAccessibilityLabel
+        }()
```

Comment:
if the accessibiliylabel is nil, why would we add ", "?  Seems like this code should be something like this:
```
        button.accessibilityHelperView.accessibilityLabel = {
            if let badgeText {
                if let accessibilityLabel = paymentMethod.paymentSheetAccessibilityLabel {
                    return "\(accessibilityLabel), \(badgeText)"
                } else {
                    return "\(badgeText)"
                }
            }
            return paymentMethod.paymentSheetAccessibilityLabel
        }()
```